tensornetwork/backends/tensorflow/tensorflow_backend.py

```diff
@@ -430,3 +430,14 @@ def eps(self, dtype: Type[np.number]) -> float:
       float: Machine epsilon.
     """
     return tf.experimental.numpy.finfo(dtype).eps
+
+  def pinv(self, tensor: Tensor, hermitian: bool = False) -> Tensor:
+    """    
+    Args:
+     tensor: A tensor.
+     rcond: Cutoff for small singular values.
+     hermitian(optional): If True, matrix provided is assumed to be Hermitian (symmetric if real-valued). Defaults to False.
+    Returns:
+     tensor: The pseudo inverse of tensor.
+    """
+    return np.linalg.pinv(tensor)
```

Comment:
you are using numpy. You should use tensorflow

Reply:
indeed